src/__tests__/utils-test.ts

```diff
@@ -181,3 +182,30 @@ test('notify page loaded', cb => {
         cb();
     });
 });
+
+test('request A/B testing configuration', cb => {
+    const AB_TESTING_CONFIGURATION = {
```

Comment:
please, rename to REMOTE_CONFIG, REMOTE_CONFIGURATION, CONFIG... (anything not related to AB testing)